zql/docs/processors/README.md

```diff
@@ -152,7 +152,7 @@ conn  1521911720.607695 CpjMvj2Cvj048u6bF1 10.164.94.120 39169     10.47.3.200 8
 |                           |                                                 |
 | ------------------------- | ----------------------------------------------- |
 | **Description**           | Add/update fields based on the results of a computed expression |
-| **Syntax**                | `put <field> = <expression>`                    |
+| **Syntax**                | `put <field> = <expression> [, <field> = <expression> ...]`     |
```

Comment:
explain the evaluation model, in particular  that clauses do not observe the results of other clauses?